blocks/rich-text/index.js

```diff
@@ -441,6 +460,60 @@ export class RichText extends Component {
 		};
 	}
 
+	getInsertionPosition() {
+		const container = this.getContainerNode();
+		const marginLeft = window.getComputedStyle( container )[ 'margin-left' ];
```

Comment:
Not sure if this could ever error in any way, but it might be prudent to use e.g. Lodash's `get` here.

Reply:
I agree, that would be safer. Made the change in https://github.com/WordPress/gutenberg/pull/5794/commits/9142f64e98f8a07b6c11b8ff4dd7acf9e092ccc0.